### PR TITLE
refactor(frontend): commnity navbar- & unauthenticated b-gradido styles

### LINKED ISSUE #2685: 🔧 [Refactor] Desktop Sub Menu
## 🔧 Refactor ticket
- Active color "#F5F5F5 inactive color "#383838"
- Max width 400px
- Replace icons
- Font-family: Work Sans Semi Bold;
- Shadow background

___

- Aktive color „#F5F5F5 inaktive color „#383838“
- Max width 400px
- Icons ersetzen
- Font-family: Work Sans Semi Bold;
- Schatten hinterlegen

image

### LINKED ISSUE #2738: 🐛 [Bug] login button not fixed size
## 🐛 Bugreport
The login button has gotten away at certain display sizes. please set it back to a clean width. 

image

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
commnity navbar style refactor

### Issues
- fixes #2685 
- fixes #2738 

https://user-images.githubusercontent.com/1324583/221109061-fe76dcf5-4724-4a33-aa1c-a60555ebc561.mp4

### CODE REVIEW

**Comment by ogerly** [comment]
https://user-images.githubusercontent.com/1324583/220915834-ad587c3f-231a-4cd0-bcf6-1bf06643e227.mp4

File: frontend/src/pages/Login.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
             </b-col>
           </b-row>
           <b-row>
-            <b-col cols="12" lg="4">
+            <b-col cols="12" lg="6">
```
Comment (ulfgebhardt) [inline frontend/src/pages/Login.vue:33]: How is this change related to this? Does this addresses our login button problem?

image

**Review by ulfgebhardt — COMMENTED** [review]
Why is `write.svg` white while the other two are black - will this cause trouble?

I like the change you show in your video! It looks really cool - lets have this merged asap!

File: frontend/src/pages/Login.vue (hunk 1 above)
Comment (ogerly) [reply frontend/src/pages/Login.vue:33]: this is not related. but i fix the problem with the login submit button with the change.

Comment (ogerly) [reply frontend/src/pages/Login.vue:33]: #2738

Comment (ogerly) [reply frontend/src/pages/Login.vue:33]: https://github.com/gradido/gradido/pull/2732

**Review by Elweyn — CHANGES_REQUESTED** [review]
- Change information button (Green => Gray)
- Registration button same as login button?

File: frontend/src/pages/Register.vue [hunk 2]
```diff
@@ -81,12 +81,13 @@
...
           </b-row>
           <b-row>
-            <b-col cols="12" lg="5">
+            <b-col cols="12" lg="6">
               <b-button
                 block
                 type="submit"
                 :disabled="disabled"
                 :variant="disabled ? 'gradido-disable' : 'gradido'"
+                class="default-class"
```
Comment (Elweyn) [inline frontend/src/pages/Register.vue:90]: Do we need this?
```suggestion
```

File: frontend/src/pages/Login.vue [hunk 3]
```diff
@@ -127,3 +127,13 @@ export default {
...
 </script>
+<style scoped>
+.btn-gradido {
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.btn-gradido-disable {
+  padding-right: 0px;
+  padding-left: 0px;
+}
```
Comment (Elweyn) [inline frontend/src/pages/Login.vue:138]: It seems we don't use this on login submit button
```suggestion
```

Or add disable logic & line 34:
add:
```
:variant="disabled ? 'gradido-disable' : 'gradido'"
```

File: frontend/src/pages/ForgotPassword.vue [hunk 4]
```diff
@@ -88,3 +90,13 @@ export default {
...
 </script>
+<style scoped>
+.btn-gradido {
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.btn-gradido-disable {
+  padding-right: 0px;
+  padding-left: 0px;
+}
```
Comment (Elweyn) [inline frontend/src/pages/ForgotPassword.vue:101]: Not used in this form:
```suggestion
```

Or add disabled logic & line 18:
```
:variant="disabled ? 'gradido-disable' : 'gradido'"
```

**Review by Elweyn — CHANGES_REQUESTED** [review]

**Review by Elweyn — APPROVED** [review]
